docs/CONTRIBUTING.md

```diff
@@ -669,126 +669,82 @@ Hello, World!
 You can start [IDE](https://github.com/enso-org/enso/tree/develop/gui) with a
 development version of the language server. IDE executable has
 `--external-backend` flag that switches off the bundled backend. That requires
-you to run the project manager process yourself. You can either get a project
-manager from one of the latest releases on
-[GitHub](https://github.com/enso-org/enso/releases), or build one using SBT
-`buildProjectManagerDistribution` command.
-
-Running development version of the IDE is possible via the `./run` script in the
-root of the repository:
+you to run the project manager process yourself. Running development version of
+the IDE is also possible via the `./run` script in the root of the repository:
 
 ```bash
-$ ./run ide start --wasm-profile dev --external-backend
+enso$ ./run gui watch --skip-wasm-opt
```

Comment:
This is the command I am using most frequently. What do other use? The original `./run ide start ...`?